src/main/java/cubicchunks/CustomCubicChunksWorldType.java

```diff
@@ -44,10 +45,10 @@ public CustomCubicChunksWorldType() {
 		ServerCubeCache cubeCache = world.getCubeCache();
 		
 		// init the worldgen pipeline
-		GeneratorStage terrain = new GeneratorStage("terrain");
-		GeneratorStage surface = new GeneratorStage("surface");
-		GeneratorStage features = new GeneratorStage("features");
-		GeneratorStage population = new GeneratorStage("population");		
+		GeneratorStage terrain = new IndependentGeneratorStage("terrain");
```

Comment:
That's understandable. For now I will just test the whole thing and if I don't find any major issues - merge it.